Makefile

```diff
@@ -13,10 +13,17 @@ include Makefile.global
 
 all: extension
 
+#build columnar only
+columnar:
+	$(MAKE) -C src/backend/columnar all
+	cp $(citus_abs_srcdir)/src/backend/columnar/columnar.so $(citus_abs_srcdir)/src/backend/distributed/
```

Comment:
Maybe there's a better way to use `columnar.so` from where it is, rather than copying it?